packages/Interface/src/DTK_ParallelTraits.hpp

```diff
@@ -25,99 +25,34 @@
 namespace DataTransferKit
```

Comment:
This header make little sense to me.  It is only pulling few Kokkos types into the DTK namespace.

Reply:
I agree. my next interface refactor will likely remove this and clean up some.

Reply:
Why postponing?

Reply:
I figured this PR was big enough with the clear purpose of introducing the map interface. The next PR would make more infrastructure changes to the C interface.

Reply:
Ok.  Would you open an issue to capture this?  Get rid of `DTK_ParallelTraits.hpp` and probably remove "imports" of Cuda/OpenMP/Serial into DTK's namespace.

Reply:
See #420